spec/02-integration/08-status_api/01-core_routes_spec.lua

```diff
@@ -40,7 +40,41 @@ describe("Status API - with strategy #" .. strategy, function()
       assert.is_number(json.server.connections_writing)
       assert.is_number(json.server.connections_waiting)
       assert.is_number(json.server.total_requests)
+      assert.is_nil(json.server.configuration_hash) -- no hash in DB more, or in DBLESS mode until configuration has been applied
     end)
+
+    it("/status starts providing a configuration_hash once an initial configuration has been pushed in dbless mode #off", function()
+      -- push an initial configuration so that a configuration_hash will be present
+      local postres = assert(client:post("/config", {
+        body = {
+          _format_version = "1.1",
+          services = ( { protocol = "http", host = "konghq.com" } ),
+        },
+        headers = { ["Content-Type"] = "application/json" },
+      }))
+      assert.res_status(201, postres)
+
+      -- verify the status endpoint now includes a value (other than the default) for the configuration_hash
+      local res = assert(client:send {
+        method = "GET",
+        path = "/status"
+      })
+      local body = assert.res_status(200, res)
+      local json = cjson.decode(body)
+      assert.is_table(json.database)
+      assert.is_table(json.server)
+      assert.is_boolean(json.database.reachable)
+      assert.is_number(json.server.connections_accepted)
+      assert.is_number(json.server.connections_active)
+      assert.is_number(json.server.connections_handled)
+      assert.is_number(json.server.connections_reading)
+      assert.is_number(json.server.connections_writing)
+      assert.is_number(json.server.connections_waiting)
+      assert.is_number(json.server.total_requests)
+      assert.is_string(json.server.configuration_hash)
+      assert.equal(32, #json.server.configuration_hash)
```

Comment:
Add assert: not equal to string.rep("0", 32)

Reply:
```suggestion
      assert.equal(32, #json.server.configuration_hash)
      assert.not_equal(string.rep("0", 32), json.server.configuration_hash)
```